.github/workflows/manifests.yml

```diff
@@ -22,11 +22,17 @@ jobs:
     runs-on: ubuntu-latest
     env:
       PYTHON_VERSION: 3.7
+      JDK_VERSION: 14
```

Comment:
If we are not backporting java 11 to 1.0 branch, can we start with 11 here and make a separate job for the 1.0 manifests?

Reply:
Everything is still on 14, no?